analyzere/extras/visualizations.py

```diff
@@ -0,0 +1,272 @@
+import hashlib
+
+from analyzere import LayerView
+from datetime import datetime
+from graphviz import Digraph
+from IPython.display import FileLink
```

Comment:
I think I need to re-think the API of the class.  Having a `view` and `download` method outside the context of a notebook don't make a lot of sense.  If I could use the notebook flag to change toggle the methods, then a `generate` method for non-notebook use cases might make more sense, and simply have it return the filenpath.